athena-gcs/src/test/java/com/amazonaws/athena/connectors/gcs/integ/GcsIntegTest.java

```diff
@@ -0,0 +1,263 @@
+import org.testng.annotations.Test;
+import software.amazon.awscdk.core.App;
+import software.amazon.awscdk.core.Stack;
+import software.amazon.awscdk.services.iam.Effect;
+import software.amazon.awscdk.services.iam.PolicyDocument;
+import software.amazon.awscdk.services.iam.PolicyStatement;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class GcsIntegTest extends IntegrationTestBase
+{
+    private static final Logger logger = LoggerFactory.getLogger(GcsIntegTest.class);
+    private final String lambdaFunctionName;
+    private static final String TABLE_NAME = "table_name";
+
+    private static final TestConfig testConfig;
+    private static final String schemaName;
+    private static final String tableName2;
+    private static final String tableName4;
+    private static final String tableName5;
+    private static final String tableName;
+    private static String lambdaAndSchemaName;
+
+    static {
+        testConfig = new TestConfig();
+        tableName = testConfig.getStringItem(TABLE_NAME).get();
+        schemaName = testConfig.getStringItem("schema_name").get();
+        tableName2 = testConfig.getStringItem("table_name2").get();
+        tableName4 = testConfig.getStringItem("table_name4").get();
+        tableName5 = testConfig.getStringItem("table_name5").get();
+    }
+
+    /**
+     * Sets up the access policy for the Lambda connector to multiple connector-specific AWS services (e.g. DynamoDB,
+     * Elasticsearch etc...)
+     *
+     * @return A policy document object.
+     */
+    @Override
+    protected Optional<PolicyDocument> getConnectorAccessPolicy()
+    {
+
+        List<String> statementActionsPolicy = new ArrayList<>();
+        statementActionsPolicy.add("*");
+        return Optional.of(PolicyDocument.Builder.create()
+                .statements(ImmutableList.of(PolicyStatement.Builder.create()
+                        .actions(statementActionsPolicy)
+                        .resources(ImmutableList.of("*"))
+                        .effect(Effect.ALLOW)
+                        .build()))
+                .build());
+    }
+
+    /**
+     * Sets the environment variables for the Lambda function.
+     */
+    @Override
+    protected void setConnectorEnvironmentVars(final Map environmentVars)
+    {
+        // environmentVars.putAll(this.environmentVars);
+    }
+
+    @Override
+    protected void setUpTableData()
+    {
+        // No-op.
+    }
+
+    /**
+     * Sets up connector-specific Cloud Formation resource.
+     *
+     * @param stack The current CloudFormation stack.
+     */
+    @Override
+    protected void setUpStackData(final Stack stack)
+    {
+        // No-op.
+    }
+
+    @BeforeClass
+    @Override
+    protected void setUp() throws Exception
+    {
+        try {
+            super.setUp();
+        }
+        catch (Exception e) {
+            throw e;
+        }
+    }
```

Comment:
There's no need to catch and then rethrow

Reply:
Removed catch block.